Makefile

```diff
@@ -85,6 +85,11 @@ test-all:
 	@echo Running all tests
 	$(GO) test -mod=vendor -race -v -tags 'unit e2e' $(GO_PACKAGES)
 
+.PHONY: coverage
```

Comment:
We can add this PHONY target to the top of the file

Reply:
I like (2/5) having the `PHONY` next to the target as this makes it clear that every target as a `PHONY`. But I would change it, ff you feel strong about this.

Reply:
I like consistency, my proposal is to put it at the top, and then, we can add another PR moving all .PHONY tags aside the tasks itself, but as part of another change.

Reply:
Opened https://github.com/mattermost/mmctl/issues/136